test/box-luatest/box_cfg_env_test.lua

```diff
@@ -166,11 +166,7 @@ g.test_uri_list = function(g)
 
```

Comment:
Nit: test-run update commits lists luatest changes, but there are several test-run's ones. I would mention them too for the sake of completeness.

Reply:
Yep, I mistakenly mentioned luatest's commits instead of test-run's. Fixed to next in commit message:

```
    Bump test-run to new version with the following improvements:

    - Bump luatest to 1.0.1-14-gdfee2f3 [1]
    - Adjust test result report width to terminal size [2]
    - dispatcher: lift pipe buffer size restriction [3]
    - flake8: fix E721 do not compare types [4]
```